studio/src/aem/aem.js

```diff
@@ -696,6 +741,219 @@ class AEM {
         }
     }
 
+    /**
+     * Creates an empty variation fragment in the target locale folder.
+     * The variation starts with no content fields - it inherits from parent at runtime.
+     * @param {Object} parentFragment - The parent fragment to create a variation from
+     * @param {string} targetLocale - Target locale for the variation (e.g., 'en_GB')
+     * @returns {Promise<Object>} The newly created empty variation fragment
+     */
+    async createEmptyVariation(parentFragment, targetLocale) {
```

Comment:
nit: I would reserve aem.js for pure fragment CRUD operations (https://developer.adobe.com/experience-cloud/experience-manager-apis/api/stable/sites/), and move this kind of M@S specific operations in mas-repository.js